covbot.py

```diff
@@ -597,6 +599,25 @@ def _short_location(self, location: str, length=int(12)) -> str:
         c.format = "org.matrix.custom.html"
         await e.respond(c, markdown=True, allow_html=True)
 
+    @command.new('risks', help=HELP['risks'][1])
+    @command.argument("risks", pass_raw=True, required=True)
+    async def risks_handler(self, event: MessageEvent, age: int) -> None:
+        if age < 0 or age > 110:
```

Comment:
Hinting isn't enough, you need to explicitly `int(age)` if you want that.

There doesn't seem to be a way to specify an explicit type in the decorator, unless I missed it:
https://github.com/maubot/maubot/blob/master/maubot/handlers/command.py#L319

Reply:
Thank you very much ! I was wondering